plugin/plugins/mijia/mijia_api/infrastructure/http_client.py

```
@@ -0,0 +1,548 @@
+        timeout = self._config.get("DEFAULT_TIMEOUT", 30)
+        self._client = httpx.Client(
+            timeout=timeout,
+            limits=httpx.Limits(
+                max_connections=100,  # 最大连接数
+                max_keepalive_connections=20,  # 最大保持连接数
+            ),
+        )
+
+    @retry(
+        stop=stop_after_attempt(3),  # 最多重试3次
+        wait=wait_exponential(multiplier=1, min=1, max=10),  # 指数退避：1s, 2s, 4s...最多10s
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)